lib/assertions/exists.js

```diff
@@ -41,7 +41,7 @@ export default function exists(options, message) {
 
 function format(selector, num) {
   if (num === undefined || num === null) {
-    return `Element ${selector} exists`;
+    return `Element ${selector} should exist`;
```

Comment:
I'm not sure this one should be changed too 🤔

Reply:
Ya I wasn't sure. Could be convinced either way. I could see the argument for an `exists` assertion saying something **exists** :P

Reply:
I've reverted this change.